src/models/bracket_races.rs

```diff
@@ -279,46 +299,7 @@ impl BracketRace {
     update_fn! {}
 }
 
-pub fn get_current_round_race_for_player(
```

Comment:
i added the "unfinished races" search b/c it's required for RR brackets. but then it turned out that "user's unfinished races" also solves the swiss case just fine, and this "current round" filter was unnecessary. i never actually wanted "the player's current round race, _even if_ that race is already finished."